storage/innobase/handler/ha_innodb.cc

```diff
@@ -6874,9 +6887,9 @@ wsrep_store_key_val_for_row(
 			index. */
 
 			const CHARSET_INFO*	cs = NULL;
-			ulint			true_len;
-			ulint			key_len;
-			const uchar*		src_start;
+			ulint			true_len=0;
+			ulint			key_len=0;
+			const uchar*		src_start=nullptr;
 			int			error=0;
 			enum_field_types	real_type;
```

Comment:
We do not need `true_len` this early at all. The body of `if (part_is_null)` right below this could simply refer to `key_len`.